Earthfile

```diff
@@ -31,9 +31,10 @@ all-arm:
   BUILD +arm-image
 
 go-deps:
-    FROM golang
+    FROM golang:1.17
```

Comment:
small nit: this sounds a good candidate for a top level `ARG`, so one place to bump the golang version used

Reply:
update: addressing to those issues in a changes subset, I see this makes currently c3os fail building as go 1.19 was released and incompatible with the codebase. I'll try to open up a separate PR with bunch of fixes I had for #47 , including this

Reply:
See https://github.com/c3os-io/c3os/pull/56

Reply:
I agree with the common arg but ill leave that for your pr and remove it here.